sdk/src/CrashReporter.ts

```diff
@@ -200,6 +214,52 @@ export default class CrashReporter {
       return;
     }
 
+    const stack = await this.cleanStackTrace(error);
+
+    const payload = await this.generateCrashReportPayload(error, stack);
+    payload.Details.Tags = getCurrentTags().concat("UnhandledError");
+
+    if (isFatal) {
+      payload.Details.Tags = getCurrentTags().concat("Fatal");
+    }
+
+    this.managePayload(payload);
+  }
+
+  /**
+   * Processes a manually sent error (using local tags).
+   * @param error - The Error to be processed.
+   * @param details
+   */
+  async processManualCrashReport(error: Error, details?: ManualCrashReportDetails) {
+    if (!error || !error.stack) {
+      warn('Unrecognized error occurred');
+      return;
+    }
+
+    const stack = await this.cleanStackTrace(error);
+
+    const payload = await this.generateCrashReportPayload(error, stack);
+
+    const payloadWithLocalParams: CrashReportPayload = {...payload};
+
+    if(details){
+      if(details.customData){
+        payloadWithLocalParams.Details.UserCustomData = Object.assign(this.customData ? this.customData : {}, details.customData);
```

Comment:
I believe this will assign the values inside of `details.customData` into the `this.customData` global data object, which is undesired as all future errors will then have this custom data.